### ISSUE TITLE
TestChannel.test_has_user fails in release -10.0

### ISSUE THREAD

**Issue by jaraco** [jaraco/irc#47]
under all pythons

``` python
======================================================= test session starts =======================================================
platform linux -- Python 3.3.5 -- py-1.4.25 -- pytest-2.6.3
plugins: cov
collected 20 items

irc/tests/test_bot.py ......F..
irc/tests/test_client.py ......
irc/tests/test_schedule.py .....

============================ FAILURES =============================
__________________ TestChannel.test_has_user ____________________

self = <irc.tests.test_bot.TestChannel object at 0x7f12c32d9d10>

    def test_has_user(self):
        channel = irc.bot.Channel()
        channel.add_user('tester1')
>       assert channel.has_user('Tester1')
E       assert <bound method Channel.has_user of <irc.bot.Channel object at 0x7f12c32d9610>>('Tester1')
E        +  where <bound method Channel.has_user of <irc.bot.Channel object at 0x7f12c32d9610>> = <irc.bot.Channel object at 0x7f12c32d9610>.has_user

irc/tests/test_bot.py:51: AssertionError
========= 1 failed, 19 passed in 3.23 seconds ============
```

Can you replicate> Do you require any further info?

---
- Bitbucket: https://bitbucket.org/jaraco/irc/issue/47
- Originally reported by: [idella5](http://bitbucket.org/idella5)
- Originally created at: 2014-11-06T09:41:38.697

**Comment by jaraco** [comment]
Update override of transform_key (using different method name). Fixes #47.

→ <<cset 2074d84fa635>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Thanks for the report. Fix released in 10.0.1.

---

Original comment by: Jason R. Coombs
